server/models/Profile.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema(
+    {
+        firstname: {
+            type: String,
+            required: true,
+        },
+        lastname: {
+            type: String,
+            required: true,
+        },
+        gender: {
+            type: String,
+            required: true,
+        },
+        birthDate: {
+            type: Date,
+            required: true,
+        },
```

Comment:
This could be a nice opportunity to explore some date validation options: (e.g. [`min/max`](https://mongoosejs.com/docs/schematypes.html#number-validators))